tests/benchdnn/pool/pool.cpp

```diff
@@ -147,6 +147,15 @@ void skip_unimplemented_prb(const prb_t *prb, res_t *res) {
         res->reason = skip_reason::case_not_supported;
         return;
     }
+
+    // There is a case that fails on intel gpu devices
+    // running sycl kernels. The blocked formats are
+    // not yet supported. We need to add them as skipped
+    if (is_gpu()) {
```

Comment:
Unconditional skipping doesn't serve the validation purpose. It must be targeted to specific known cases.